src/core/agent/services/signifyNotificationService.test.ts

```diff
@@ -812,6 +862,176 @@ describe("Signify notification service of agent", () => {
     }
     expect(markNotificationMock).toBeCalledTimes(1);
   });
+  test("Should skip if notification route is /multisig/exn when haven't accepted", async () => {
```

Comment:
Please name these appropriately. "original grant is linked to first received /multisig/exn admit message, and no notification record is created"